test/ffmpeg-qsv/encode/encoder.py

```diff
@@ -167,7 +169,10 @@ def check_output(self):
     assert m is not None, "It appears that the QSV plugin did not load"
 
     if vars(self).get("ladepth", None) is not None:
-      m = re.search("Using the VBR with lookahead \(LA\) ratecontrol method", self.output, re.MULTILINE)
+      if "vbr" == self.rcmode:
+        m = re.search("Using the VBR with lookahead \(LA\) ratecontrol method", self.output, re.MULTILINE)
+      elif "icq" == self.rcmode:
+        m = re.search("RateControlMethod: LA_ICQ", self.output, re.MULTILINE)
```

Comment:
What about ICQ without LA?

Would it be better to search for rate control messages generated by the qsvenc.c select_rc_mode function instead (https://github.com/intel-media-ci/ffmpeg/blob/b414cff6304071d5ef1e2b0a1b407e4fe3f58092/libavcodec/qsvenc.c#L373)?

Reply:
Yes, of course. PR update, please review , thanks.